src/components/CopyButton/CopyButton.test.js

```diff
@@ -34,21 +34,21 @@ it('displays ✔ on click', () => {
   const tree = render(<CopyButton text="testing" />);
 
   // There should be no ✔ yet, only an Icon
-  expect(tree.queryByText('✔')).toBeNull();
+  expect(tree.container.querySelector('span.CopyButton')).toBeNull();
   expect(tree.container.firstChild).toMatchSnapshot();
 
   let button = tree.container.querySelector('button');
   button.click();
 
   // Icon should be replaced with ✔
```

Comment:
Can you update this comment now that the behavior has been changed and there is no longer a ✔ character?